.github/workflows/test-linux.yml

```diff
@@ -68,6 +77,12 @@ jobs:
             PYTHON_VERSION: '3.8'
     timeout-minutes: 90
     steps:
+      - name: Setup Remote SSH Connection
+        if: env.ENABLE_SSH == 'true'
+        uses: mxschmitt/action-tmate@v3
+        timeout-minutes: 10
+        with:
+          detached: true
```

Comment:
Is this what prevents blocking the workflow at this point and allows to run the other steps? Just curious.

Reply:
Yes, that is correct.